pydash/pydash_app/dashboard/aggregator/__init__.py

```diff
@@ -0,0 +1,104 @@
+from ordered_set import OrderedSet
+from collections import OrderedDict
+
+import persistent
+
+from . import statistics
+
+
+def date_dict(dict):
+    # JS expects dates in the ISO 8601 Date format (example: 2018-03)
+    return {k.strftime("%Y-%m-%d"): v for (k, v) in dict.items()}
+
+class Aggregator(persistent.Persistent):
+    """
+    Maintains aggregate data for either a dashboard or a single endpoint.
+    This data is updated every time a new endpoint call is added.
+    """
+
+    contained_statistics_classes = OrderedSet([
```

Comment:
The Aggregator-class can be splitted into multiple subclasses. (And inherit the Aggregator-class).

Reply:
This `contained_statistics_classes` is just a collection of class references to statistics we want to aggregate, ehile the actual definition of these classes are defined in `statistics.py`.
I have no idea how you would want to split `Aggregator` up, since it just keeps a number of `Statistic`s instances, feeds them the endpoint-calls it retrieves and dequests their formatted values when required.